native/src/routes/Pois.tsx

```diff
@@ -118,7 +131,14 @@ const Pois = ({
           navigateToFeedback={createNavigateToFeedbackForPoi(poi)}>
           <>
             {location && (
-              <PageDetail identifier={t('location')} information={location} theme={theme} language={language} />
+              <PageDetail
+                identifier={t('location')}
+                information={location}
+                theme={theme}
+                language={language}
+                linkLabel={poi?.featureLocation && t('map')}
+                onLinkClick={navigateToMap(cityModel.code, language, String(poi.location.id))}
```

Comment:
A more general question/discussion: I think for UX it is better to have speaking ids in the urls, i.e. to use the poi slug instead of the id, e.g. `agentur-fuer-arbeit-augsburg` instead of `12`. We also did it like that whenever possible in other routes. What do you think?

Reply:
i mean i don't see the advantages to be honest for mobile? Maybe we can discuss this tomorrow after weekly

Reply:
Question was mostly for web, but its relevant on mobile as well, e.g. for deep linking. Sadly I won't have time for that as I am already on vacation then :/

Reply:
I suggest to make a separate ticket for that